### ISSUE TITLE
Sennheiser ADN CU1

### ISSUE THREAD

**Issue by Cedicedi** [bitfocus/companion-module-requests#1095]
Hello
I'm using a Sennheiser ADN CU1 Conference System, which is connected with up to 40 microphones over ethernet.
Is it possible to make a module, which give a feedback about which microphone is unmuted?
I'm exited for your answers.
Greetings,
Cedi

**Comment by fbosman** [comment]
Hi,Do you have documentation about the api or commands  available?Met vriendelijke groet,Frans BosmanVerstuurd vanaf mijn iPhoneOp 21 mrt. 2023 om 17:06 heeft Cedicedi ***@***.***> het volgende geschreven:﻿
Hello
I'm using a Sennheiser ADN CU1 Conference System, which is connected with up to 40 microphones over ethernet.
Is it possible to make a module, which give a feedback about which microphone is unmuted?
I'm exited for your answers.
Greetings,
Cedi

**Comment by Cedicedi** [comment]
Here's the link to the product:
https://de-ch.sennheiser.com/konferenz-tagung-steuerung-sprechstellen-diskussions-30-40-adn-cu1

And here should be the informations about the protocol:
[QG_ADNMediaCtrlProtocol_A02_EN.pdf](https://github.com/bitfocus/companion-module-requests/files/11037468/QG_ADNMediaCtrlProtocol_A02_EN.pdf)

I hope this will help :)

**Comment by Cedicedi** [comment]
Hello
Is it possible to speed up the development of the module by offering a financial reward? :)
Greetings,
Cedi

**Comment by fbosman** [comment]
Hi,
There can be up to 40 mics connected. Do you want feedback to 40 buttons, or 40 variables or one variable with a row of muted mics?
Can you pm me so we can have a conversation?

**Comment by leonreucher** [comment]
Still required?

**Comment by fbosman** [comment]
I'm working on this module

**Comment by leonreucher** [comment]
Ah okay, perfect.

**Comment by Nico380** [comment]
Hi,
im also in touch with Sennheiser ADN. What's about the Plugin? It is working?

Greetings!

**Comment by Cedicedi** [comment]
> Hi, im also in touch with Sennheiser ADN. What's about the Plugin? It is working?
> 
> Greetings!

Hi,

The development was stopped. However, the module works with certain restrictions.
The miks cannot be freely numbered.
For more information send me a pm.

**Comment by zdeezr** [comment]
Hi,

I am unable to send a PM, but I am interested in this module.
The main reason is the feedback on the mic status.

Even though the development is incomplete, does this module provide any feedback?

**Comment by fbosman** [comment]
Yes, it does, however, you cannot use this module and the sennheiser controller together. It is one of the other. That’s the reason the development is on hold.Met vriendelijke groet,Frans BosmanOp 20 mrt 2025 om 17:05 heeft zdeezr ***@***.***> het volgende geschreven:﻿
  Hi,
I am unable to send a PM, but I am interested in this module.
The main reason is the feedback on the mic status.
Even though the development is incomplete, does this module provide any feedback?—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>



zdeezr left a comment (bitfocus/companion-module-requests#1095)
Hi,
I am unable to send a PM, but I am interested in this module.
The main reason is the feedback on the mic status.
Even though the development is incomplete, does this module provide any feedback?

**Comment by Nico380** [comment]
Thx for the Feedback! How can we get the Module for using it in companion?
